packages/framework/src/Form/Admin/Order/OrderPaymentFormType.php

```diff
@@ -53,6 +69,18 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'data_class' => OrderItemData::class,
                 'attr' => ['novalidate' => 'novalidate'],
+                'validation_groups' => function (FormInterface $form) {
```

Comment:
I hear often about violation the DRY principle from colleagues... So here, you are violating the DRY principle. I personally don't care in this case, but the team may ;-)

Reply:
The block of code is the same in 3 form types

Reply:
You're right about the DRY violation, but resolving validation groups is always done in an inlined function like this, so I was just trying to be consistent. I guess I could extract it into a public static function in `OrderItemFormType` - do you think it would be better?

Reply:
As I said "I personally don't core in this case". So it's up to you. I only wanted to point it out. Maybe you can explain your decision in the commit message

Reply:
Well, there was quite a lot of duplicated code because of this, I did the extraction in 6f8c26436